dlt/destinations/impl/filesystem/filesystem.py

```diff
@@ -167,31 +178,53 @@ def initialize_storage(self, truncate_tables: Iterable[str] = None) -> None:
                         " should be created previously!"
                     )
 
+        # we mark the storage folder as initialized
+        self.fs_client.touch(posixpath.join(self.dataset_path, INIT_FILE_NAME))
```

Comment:
unfortunately when dropping the azure dataset, the folders still remain, only the files will be gone, so we need a marker to see wether the dataset is initialized..